### PR TITLE
User guide, v.2

### PR DESCRIPTION
**Pull request by Maria-Alekseeva**

### CODE REVIEW

File: source/guide/using-genestack-public-data.inc [hunk 1]
```diff
@@ -0,0 +1,92 @@
+.. _public-experiment-label:
```
Comment (klas-genestack) [inline source/guide/using-genestack-public-data.inc]: Why do you call these files "*.inc"?

File: source/guide/pipelines-bisseq.rst [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+.. image:: images/bisulfite_seq_mapping_app_page.png
+
+Let’s talk a bit about various settings:
+
+1. The **Number of mismatches** option lets you set the maximum number of allowed
+   mismatches per read. Changing this number you can affect application
+   runtime and percentage of mapped reads. There is an increase in the
+   percentage of mapped reads and in the application runtime when increasing
+   this value. For example, by default the read could be mapped to the genome
+   with no more than 5 mismatches. (default: 5)
```
Comment (Maria-Alekseeva) [inline source/guide/pipelines-bisseq.rst]: When we talk about options, we always use this punctuation for default values. I would use one style everywhere, so let's leave it at that so far. If we come up with some better solution, we could have implemented it by the upcoming release. I'll make a note for that.

**Comment by klas-genestack** [comment]
I have a *huge* problem with file layout of guide, though.  Let's take a look at sources:
```
 % ls source/guide/                        
images                                 pipelines-bisseq_include.rst                  pipelines-ref-genomes_include.rst             using-genestack-building-pipelines_include.rst  using-genestack-run-DF_include.rst
genestack-features.rst                 pipelines-expression_arrays_include.rst       pipelines-rnaseq_include.rst                  using-genestack-export_include.rst              using-genestack-sharing_include.rst
getting-help.rst                       pipelines-mappedQC-preprocessing_include.rst  pipelines-variants_preprocessing_include.rst  using-genestack-import_include.rst              using-genestack-signup-dashboard.rst
intro-to-bioinformatics.rst            pipelines-methyl_arrays_include.rst           pipelines-wgs-and-wes_include.rst             using-genestack-initialization_include.rst      using-genestack-task-manager_include.rst
intro-to-ngs.rst                       pipelines-microbiome_include.rst              references.rst                                using-genestack-public-data_include.rst
pipelines-additional_apps_include.rst  pipelines-rawQC-preprocessing.rst             using-genestack-browse-data_include.rst       using-genestack-public-DF_include.rst
```
It's a tricky job even guessing how many result documents are there, let alone how they are organized.
I think there should be one "index" file here (like `pipelines.rst`) and a bunch of related includes in subdir (`pipelines_include/microbiome.rst`, `pipelines_include/wgs-and-wes.rst` etc).  Or at the very least files should be named more explicitly.
But this is not a subject of this PR, of course.

**Comment by klas-genestack** [comment]
> it's good to go into production after #41 is merged

Speaking of production: while I'm away, Andrey could do it no trouble.

**Pull request merged**